### ISSUE TITLE
Ring gems aren't exported while looking at vault

### ISSUE THREAD

**Issue by iXerxes** [simulationcraft/simc-addon#30]
If you're looking at the great vault rewards and generate a SimC report, gems in rings don't get exported.

Generate looking at vault: https://hastebin.com/axakepudez
Generate normal: https://hastebin.com/uwekuvuzuh

**Comment by seriallos** [comment]
Hmm, that's definitely weird.

The first export isn't showing any gear in your bags - did you run /simc immediately after logging in or were your bags actually empty?

I've never been able to reproduce it myself, but I believe there's a rare issue where /simc can miss some information about your character if it's run shortly after login.

It could be something else but I'm not seeing widespread problems with gems missing from Vault sims. I did a spot check of a few dozen sims today and I'm seeing a bunch of /simc input that has Vault gear and ring gems.

**Comment by iXerxes** [comment]
I logged in, and around 5-10m after running around Oribos, I went to my vault. The export window was opened using the minimap icon, but I doubt that means anything.

As I'm write this, I remembered I had another vault that I could test with and I can't replicate it either.

One thing I should note though, is that this time around, I _did_ inspect my character, go through multiple zones and also open my bags. Maybe this has something to do with it, as I don't remember doing any of those the first time.

This is probably just a rare, insignificant bug, but I thought I'd report it anyway since I thought at the time I didn't have other characters to test with.
